requirements/base.txt

```diff
@@ -1,32 +1,11 @@
-argparse==1.4.0
-behave==1.2.6
-decorator==4.4.0
-django-appconf==1.0.3
-django-ckeditor==5.7.1
-django-compressor==2.3
-django-debug-toolbar==2.0
-django-extensions==2.2.1
-django-logentry-admin==1.0.4
-django-registration-redux==2.6
-django-uwsgi==0.2.2
-Django==2.2.3
-djangocms-admin-style==1.4.0
-factory-boy==2.12.0
-ipdb==0.13.9
-ipython-genutils==0.2.0
-ipython==7.32.0
-path.py==12.0.1
-pexpect==4.7.0
-pickleshare==0.7.5
-Pillow==9.0.1
-python-dateutil==2.8.0
+django==4.0.3
+django-ckeditor==6.2.0
+django-compressor==3.1
+django-debug-toolbar==3.2.4
+django-extensions==3.1.5
+django-logentry-admin==1.1.0
+django-registration-redux==2.10
+git+https://github.com/Doca/django-ajax.git#egg=djangoajax
+
+factory-boy==3.2.1
```

Comment:
Is it really "base" or only "dev"?

Reply:
`factory-boy` is used in the `create_dummy_data` management command (and tests, but that's ). Currently all management commands are deployed to production... this makes this a requirement for all environments, I guess? However, we could remove it from base and move it to dev, in order to preven accidential execution of the command on a production environment. 

In the future: to make specific commands unavailable in a certain environment, we could collecte debugging and development commands in a new app (eg. `devtools`), and add it to `INSTALLED_APPS` only in non-production settings.